packages/client/src/v2-events/components/forms/utils.ts

```diff
@@ -129,3 +145,36 @@ export function unflatten<T>(
 
   return result
 }
+
+const initialValueMapping: Record<FieldType, IFormFieldValue | null> = {
+  [FieldType.TEXT]: INITIAL_TEXT_VALUE,
+  [FieldType.DATE]: INITIAL_DATE_VALUE,
+  [FieldType.RADIO_GROUP]: INITIAL_RADIO_GROUP_VALUE,
+  [FieldType.PARAGRAPH]: INITIAL_PARAGRAPH_VALUE,
+  [FieldType.FILE]: null,
+  [FieldType.HIDDEN]: null
+}
```

Comment:
Let's move these and the logic in `fieldValueToString` to the registered field component files.

```
export const initialValue = <value>
```

We should also consider if we need initial values to be definited like this at all or if all registered components should assume "undefined" as the initial empty value. I'm saying this because for instance an empty string `""` is not the same as "user didn't input anything" as cleaning they might have written something to the input and then decided to make it empty